fdbserver/ClusterController.actor.cpp

```diff
@@ -508,6 +516,9 @@ class ClusterControllerData {
 
 		id_used[masterProcessId]++;
 		id_used[clusterControllerProcessId]++;
+		if (db.serverInfo->get().distributor.isValid()) {
```

Comment:
change distributor to Optional<DataDistributionInterface> and change this to distributor.present()